src/main/scala/com/datastax/cdm/job/BaseJob.scala

```diff
@@ -70,15 +69,11 @@ abstract class BaseJob[T: ClassTag] extends App {
     maxPartition = getMaxPartition(propertyHelper.getString(KnownProperties.PARTITION_MAX), hasRandomPartitioner)
     coveragePercent = propertyHelper.getInteger(KnownProperties.TOKEN_COVERAGE_PERCENT)
     numSplits = propertyHelper.getInteger(KnownProperties.PERF_NUM_PARTS)
-    partitionFile = propertyHelper.getString(KnownProperties.PARTITIONS_TOKEN_RANGE_FILE)
```

Comment:
Is this being handled in another class?  This exception file generated from the main job run (for each table) will be the input for rerunning jobs with with the class DiffPartitionsFromFile / MigratePartitionsFrom File.

Reply:
@faizalrub-datastax This property was not used in the code anywhere. It instead uses `keyspace.tablename` to generate the file name. I have updated the logic a bit, it now generates the file as `/exception-dir/<keyspace>.<table>_partitions.csv` & this file can be used to rerun `DiffPartitionsFromFile` & `MigratePartitionsFrom` jobs. The same has been documented in the readme [here](https://github.com/datastax/cassandra-data-migrator/tree/feature/CDM-87#migrating-specific-partition-ranges). Applicable configs have also been cleaned up.

Reply:
@pravinbhat for this particular customer, job runs are automated and the exception file is passed as conf parameter for the spark-submit as follows ,  --conf spark.input.partitionFile=<dir>/<filename>

Reply:
@faizalrub-datastax However that is not used in the code anywhere including `DiffPartitionsFromFile` & `MigratePartitionsFrom`. Can you check if these changes from your customer specific branches were missed & not merged in main.